manifests/charts/istio-cni/templates/daemonset.yaml

```
@@ -38,6 +38,16 @@ spec:
     spec:
       nodeSelector:
         kubernetes.io/os: linux
+      # Allows for exluding instio-cni from being scheduled on specified nodes
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: maistra.io/exclude-cni
+                operator: NotIn
+                values:
+                - true
```

I think this manifest will fail to apply because of this boolean value. It should be a string.
```suggestion
                - "true"
```